src/main/java/org/sonarqube/A.java

```diff
@@ -1,4 +1,14 @@
 package org.sonarqube;
 
 public class A {
+  String message = "foo";
+
+  // TODO fix this
```

Comment:
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_1@6
Complete the task associated to this TODO comment.